audformat/core/utils.py

```diff
@@ -1730,6 +1728,24 @@ def _is_same_dtype(d1, d2) -> bool:
     return d1.name == d2.name
 
 
+def _maybe_convert_int_dtype(
```

Comment:
We have more or less the same code in `MiscTable.__init__()` so maybe we can call this function there, too?

Reply:
Not so easy as we need access to `levels` and `dtypes` after the code block.

Reply:
OK, I reuse the helper function now.